### PR TITLE
Extend configurability of dxp's position on screen

### LINKED ISSUE #14: Add an option to center dexpo on the monitor at runtime
Add more customization options to dexpo's location:
Prepend `-` to the coordinate to calculate it from the left

### PR DESCRIPTION
**Pull request by mmskv**
- dxp can be placed in any of the moniotr's corners
- Specify what monitor to put dxp on with `dxp_monitor_name`
- `dxp_monitor` defaults to primary monitor with `get_monitor_primary()`
- dxp can be centered both vertically and horizontally

Closes #14

### CODE REVIEW

File: src/window.cpp [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 
 constexpr bool k_horizontal_stacking = (dxp_width == 0);
 constexpr bool k_vertical_stacking = (dxp_height == 0);
 
-window::window (const int16_t x, ///< x coordinate of the top left corner
-                const int16_t y, ///< y coordinate of the top left corner
-                const std::vector<dxp_socket_desktop> &desktops)
-    : drawable (x, y, 0, 0) // Width and height will be calculated from config
+window::window (const std::vector<dxp_socket_desktop> &desktops)
+    : drawable (0, 0, 0, 0) // Width and height will be calculated from config
```
Comment (TheHussky) [inline src/window.cpp]: Do we really need first two arguments then? I think there's a better way to code it.

Comment (TheHussky) [reply src/window.cpp]: oh, the function requires it, i see. Still, those four zeros seem weird. Maybe set args by default?

File: src/xcb_util.hpp [hunk 2]
```diff
@@ -80,6 +81,13 @@ std::vector<uint32_t> get_property_value (xcb_connection_t *c,
  */
 std::vector<monitor_info> get_monitors (xcb_connection_t *c, xcb_window_t root);
 
+/**
+ * Get crtc of the primary monitor.
+ *
+ * NOTE: Returned monitor_info.name is unsed as it is unused.
```
Comment (TheHussky) [inline src/xcb_util.hpp]: it's unset (not unsed) i believe. I guess it's better rto fix it here and in a couple more places

**Review by TheHussky — APPROVED** [review]
You may want to fix minor inconveniences

File: src/xcb_util.hpp (hunk 2 above)
Comment (mmskv) [reply src/xcb_util.hpp:87]: wdym?

File: src/window.cpp (hunk 1 above)
Comment (mmskv) [reply src/window.cpp]: will do

File: src/xcb_util.hpp (hunk 2 above)
Comment (mmskv) [reply src/xcb_util.hpp]: fixed

**Review by TheHussky — APPROVED** [review]
lgtm

**Pull request merged**